homeassistant/components/zha/update.py

```diff
@@ -130,14 +130,9 @@ def __init__(
     def _get_cluster_version(self) -> str | None:
         """Synchronize current file version with the cluster."""
 
-        device = self._ota_cluster_handler._endpoint.device  # pylint: disable=protected-access
-
         if self._ota_cluster_handler.current_file_version is not None:
             return f"0x{self._ota_cluster_handler.current_file_version:08x}"
 
-        if device.sw_version is not None:
-            return device.sw_version
```

Comment:
For reference, this is the same value as `current_file_version` (this is not to be confused with `sw_build_id`), but I'm fine if we no longer use the value saved in the HA device registry. It should also be always in zigpy cache now.

Reply:
I think at some point we put `sw_build_id` or some other version *string* into the `sw_version`, as I was seeing screenshots of devices with a "latest version" like 0x12345678 but a "current version" like `1.2.3`